src/tools/opensc-explorer.c

```diff
@@ -53,6 +53,11 @@
 
 #define DIM(v) (sizeof(v)/sizeof((v)[0]))
 
+#ifndef MIN
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))
+#endif
```

Comment:
This is already defined in `src/libopensc/internal.h`. Would it work to use the definition from there, rather than redefining it all over again in each tool? Other tools already include `internal.h`.